js/ui/map.js

```diff
@@ -139,6 +139,7 @@ class Map extends Camera {
 
         if (typeof options.container === 'string') {
             this._container = window.document.getElementById(options.container);
+            if (!this._container) throw new Error(`No element found in the document with ID '${options.container}' as specified as the container in the Map constructor.`);
```

Comment:
Do you think it's possible to shorten this error message without making it less intuitive for our users?